blockstore/src/main/java/brooklyn/location/blockstore/BlockDeviceOptions.java

```diff
@@ -72,7 +77,8 @@ public static BlockDeviceOptions copy(BlockDeviceOptions other) {
     			.tags(other.tags)
     			.sizeInGb(other.sizeInGb)
     			.deviceSuffix(other.deviceSuffix)
-    			.deleteOnTermination(other.deleteOnTermination);
+                .deleteOnTermination(other.deleteOnTermination)
```

Comment:
Strange indent is because other (existing) lines use tabs! I'll fix that in a separate commit.